cristin-import/src/test/resources/features/JournalArticleRules.feature

```diff
@@ -7,6 +7,21 @@ Feature: Mapping of Journal Article
     When the Cristin Result is converted to an NVA Resource
     Then the NVA Resource is an instance of "JournalArticle"
 
+  Scenario: Map returns a Journal Article NVA Resource when the Cristin Result is an "ARTICLE"
```

Comment:
An "Academic Article" maps to a Journal Article. I think it is better if in the scenario titles we use the english name of the cristin category as it appears in the excel file (column VARBEIDUNDERKATNAVN_ENGELSK).